packages/components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -177,60 +154,42 @@ export const Tooltip = ({
                 ref={tooltipRef}
                 disabled={disabled}
                 {...rest}
-                render={(attrs, _content, instance) =>
-                    rich ? (
-                        <BoxRich
-                            $maxWidth={maxWidth}
-                            tabIndex={-1}
-                            variants={animationVariants}
-                            animate={isShown ? 'shown' : 'hidden'}
-                            transition={{ duration: 0.2, ease: 'easeInOut' }}
-                            onAnimationComplete={
-                                // @ts-expect-error
-                                isShown ? () => {} : tooltipRef.current?._tippy?.unmount //  eslint-disable-line no-underscore-dangle
-                            }
-                            {...attrs}
-                        >
-                            {title && <StyledTooltipTitle>{title}</StyledTooltipTitle>}
-                            <StyledContent>{content}</StyledContent>
-                        </BoxRich>
-                    ) : (
-                        <BoxDefault
-                            $maxWidth={maxWidth}
-                            tabIndex={-1}
-                            variants={animationVariants}
-                            animate={isShown ? 'shown' : 'hidden'}
-                            transition={{ duration: 0.2, ease: 'easeInOut' }}
-                            onAnimationComplete={
-                                // @ts-expect-error
-                                isShown ? () => {} : tooltipRef.current?._tippy?.unmount //  eslint-disable-line no-underscore-dangle
-                            }
-                            {...attrs}
-                        >
-                            {title && <StyledTooltipTitle>{title}</StyledTooltipTitle>}
-                            {guideAnchor && instance && (
-                                <OpenGuideInner>{guideAnchor(instance)}</OpenGuideInner>
-                            )}
-                            <StyledContent>{content}</StyledContent>
-                            {readMore && (
-                                <ReadMoreLink
-                                    variant="nostyle"
-                                    href={readMore.link}
-                                    target="_blank"
-                                >
-                                    {readMore.text}
-                                </ReadMoreLink>
-                            )}
-                        </BoxDefault>
-                    )
-                }
+                render={(attrs, _content, instance) => (
+                    <TooltipContainer
+                        $isLarge={isLarge}
```

Comment:
nit: `isLarge` is declared as transient here, but not on `TooltipTitle`: It does not need the `$` because it is not a string.

Reply:
17e7e4b887